examples/case_studies/bayesian_ab_testing_introduction.ipynb

```diff
@@ -1,5 +1,20 @@
 {
```

Comment:
<p>You should use a citation like the one above for the doing bayesian data analysis book. It is already on the bib file so you don't need to add it: https://github.com/pymc-devs/pymc-examples/blob/main/examples/references.bib#L249</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>